galley/pkg/config/analysis/diag/message.go

```diff
@@ -21,6 +21,8 @@ import (
 	"istio.io/istio/galley/pkg/config/resource"
 )
 
+const DocPrefix = "https://istio.io/docs/reference/config/analysis"
```

Comment:
nit: does this const need to be exported?

Reply:
It's referenced from istioctl/cmd/analyze.go, so yes.